ReactiveCocoa/WatchKit/WKInterfaceLabel.swift

```diff
@@ -13,7 +13,7 @@ extension Reactive where Base: WKInterfaceLabel {
 	}
 	
 	/// Sets the color of the text of the label.
-	public var textColor: BindingTarget<UIColor> {
+	public var textColor: BindingTarget<UIColor?> {
```

Comment:
I have changed the type to optional, to match the WatchKit original type. Is that ok to do it that way, without some kind of deprecation?

Reply:
Not really, but there's also not really a way to deprecate it. I think we'll just have to go with it.